CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape_FactionHeader.uc

```diff
@@ -77,7 +77,7 @@ simulated protected function UpdateText()
 
 	if (bIsOngoing)
 	{
-		strText = class'UIUtilities_Infiltration'.static.MakeFirstCharCapOnly(class'UICovertActions'.default.CovertActions_CurrentActiveHeader);
```

Comment:
Can you explain this change?

Reply:
Before, it was `In progress` which was really annoying. Now it's `In Progress`